docs/src/Button.doc.js

```diff
@@ -315,23 +324,92 @@ function MenuButtonExample() {
 
 card(
   <Example
+    id="accessibility-popup"
     description={`
-    We want to make sure every button on the page is unique when being read by screenreader.
-    \`accessibilityLabel\` allows us to update the spoken text.
+      We want to make sure every button on the page is unique when being read by screenreader.
+      - \`accessibilityHaspopup\` allows us to specify that the button has associated content (i.e. Flyout).
+      - \`accessibilityLabel\` allows us to update the spoken text.
+  `}
+    name="Example: Accessibility (Popup)"
+    defaultCode={`
+function A11yExPopup() {
+  const [isOpen, setOpen] = React.useState(false);
+  const anchorRef = React.useRef(null);
+
+  return (
+    <Box>
+      <Box 
+        color="lightGray"
+        display="inlineBlock" 
+        ref={anchorRef} 
+        rounding="pill" 
+      >
+        <Button
+          accessibilityHaspopup
+          accessibilityLabel="see more"
+          onClick={() => setOpen(!isOpen)}
+          inline
+          text="See more"
+        />
+      </Box>
+      {isOpen && (
+        <Flyout 
+          anchor={anchorRef && anchorRef.current} 
+          idealDirection="right"
+          onDismiss={() => undefined} 
+        >
+          <Box padding={2}>
+            <Text>I am a popup.</Text>
+          </Box>
+        </Flyout>
+      )}
+    </Box>
+  );
+}
+`}
+  />
+);
 
-    Be sure to internationalize your \`accessibilityLabel\`!
+card(
+  <Example
+    id="accessibility-disclosure"
+    description={`
+      We want to make sure every button on the page is unique when being read by screenreader.
+      - \`accessibilityControls\` allows us to specify the \`id\` of an associated content element (i.e. Accordion panel) which is controlled by this button.
```

Comment:
@themindfuldev can we remove `allows us to...` in every sentence in this PR? E.g. `accessibilityControls specifies the id...`